_data/changelogs/component-character-count.yml

```diff
@@ -2,6 +2,13 @@ title: Character count
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Enhanced visual cue when `maxlength` is exceeded.
+    summaryAdditional:  Now, the component uses standard USWDS error styles to visually enhance the error state.
+    affectsJavascript: true
```

Comment:
**question**: Should this include the "styles" tag? The related PR doesn't touch the CSS itself, but does change how the component visually displays. Flagging to hear other thoughts around this. 
```suggestion
    affectsJavascript: true
    affectsStyles: true
```

Reply:
It should according to changelog README description.

https://github.com/uswds/uswds-site/blob/1781d23208346bde7badb2c441535099675745c4/_data/changelogs/_CHANGELOG-README.md?plain=1#L16